model/lifecycle.go

```diff
@@ -845,9 +851,26 @@ func TryMarkPatchBuildFinished(b *build.Build, finishTime time.Time, updates *St
 	return nil
 }
 
+func getTaskCreateTime(project *Project, v *version.Version) (*time.Time, error) {
+	if evergreen.IsPatchRequester(v.Requester) {
+		baseVersion, err := version.FindOne(version.BaseVersionFromPatch(project.Identifier, v.Revision))
+		if err != nil {
+			grip.Error(errors.Wrap(err, "Error finding base version for patch version"))
+			return nil, err
+		} else if baseVersion == nil {
+			err = errors.Errorf("Could not find base version for patch version %s", v.Id)
+			grip.Warning(err)
```

Comment:
probably not worth logging and returning errors.

Reply:
I removed the logging for these errors.